src/audio.rs

```diff
@@ -80,7 +86,10 @@ impl Audio {
     ///
     /// The source can be created via [AudioSource::new] and modified by builders on [AudioSource]
     /// (like [AudioSource::looping]).
-    pub fn play(&mut self, source: AudioSource) {
+    pub fn play(&mut self, mut source: AudioSource) {
+        if source.gain_variance != 0.0 {
+            source.gain += random::Uniform.between(-source.gain_variance, source.gain_variance);
+        }
```

Comment:
Nice!

Maybe check if the gain is 0? Or can you change the gain as it's playing?

Reply:
There is no way to change the gain currently. That should be part of a bigger "AudioID" PR.